dalle2_pytorch/trainer.py

```diff
@@ -14,6 +14,8 @@
 from dalle2_pytorch.version import __version__
 from packaging import version
 
+from accelerate import Accelerator
```

Comment:
do we want to lazy import this? or should we depend on HFA?

Reply:
i think we should depend on it 😄 i think it should have a good future!

Reply:
drop a thank you to huggingface too! in particular Sylvain for his awesome engineering work :smile:

Reply:
love that, will do!